src/nodes/accessors/Normal.js

```diff
@@ -86,7 +86,7 @@ export const normalWorldGeometry = /*@__PURE__*/ ( Fn( ( builder ) => {
 }, 'vec3' ).once() )();
 
 /**
- * TSL object that represents the transformed vertex normal in view space of the current rendered object.
+ * TSL object that represents the vertex normal of the current rendered object in view space.
```

Comment:
The word "transformed" was removed. Is that OK, or do you want the word restored?

Reply:
If we are going to remove the term transformed, I think it would be better to modify the description of nodes with the *Geometry suffix so that it is explicit that they are not modified with skinned or morph.

Reply:
Yes, that is why I removed the term.

And I mentioned this because with this PR `normalView` and `normalViewGeometry` now have the identical descriptions -- as do `normalWorld` and `normalWorldGeometry`.

I do not know how to describe the accessors having the *Geometry suffix. With the exception of the three.js roughness calculation, I am not aware of any other use cases in the literature where one would need them.

This is why I asked if you would consider removing them, and using them only internally. That would be my preference.

Reply:
It is definitely something very specific and I think each engine assigns a name to it, Unreal uses Pre-Skinned Normal for similar purpose and gives some examples using tri-planar texture [here](https://dev.epicgames.com/documentation/en-us/unreal-engine/vector-material-expressions-in-unreal-engine). I think it is useful that we have it, as we saw this was used to optimize the code in some examples like `webgpu_tsl_earth`, `webgpu_reflection` and the background shader, so it is positive.

Reply:
OK. Do you like the Pre-Skinned terminology? I can refer to it as the "pre-skinned vertex normal of the currently-rendered object in view space."

Reply:
I’d like it to be automatic for the user to understand that the `Geometry` suffix does not process vertices for skinned, morphed, or custom transformations. `Pre-Skinned` seems to only refer to skinned meshes. Maybe we could add something like this?

```js
/**
 * TSL object that represents the vertex normal of the currently rendered object in view space.
 * 
 * Nodes with the *Geometry suffix do not process skinned, morphed, or custom vertex transformations.
 *
 * @tsl
 ```

This is present in other places as well as `positionGeometry`.